src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -52,12 +52,11 @@ const ReportActionItemCreated = (props) => {
             onClose={() => Report.navigateToConciergeChatAndDeleteReport(props.report.reportID)}
         >
             <View style={StyleUtils.getReportWelcomeContainerStyle(props.isSmallScreenWidth)}>
-                <View pointerEvents="none" style={StyleUtils.getReportWelcomeBackgroundImageViewStyle(props.isSmallScreenWidth)}>
-                    <ImageBackground
-                        source={EmptyStateBackgroundImage}
-                        style={StyleUtils.getReportWelcomeBackgroundImageStyle(props.isSmallScreenWidth)}
-                    />
-                </View>
+                <Image
+                    source={EmptyStateBackgroundImage}
```

Comment:
```suggestion
                <Image
                    pointerEvents="none"
                    source={EmptyStateBackgroundImage}
```
So that we can't select the background image (see [this comment](https://github.com/Expensify/App/pull/13647#pullrequestreview-1232638704))

Reply:
Oh really? I was able to right click and download (Chrome on mac)